core/standalone/build.gradle

```diff
@@ -37,30 +38,40 @@ scoverage {
     scoverageScalaVersion.set("${gradle.scala.scoverageScalaVersion}")
 }
 
+task downloadSwagger(type: Download) {
```

Comment:
another thought, there is likely a swagger Gradle plugin that will do this work for you. no need to migrate for the purposes of this PR, im just writing it here for learning.